### PR TITLE
Handle Simple bits on InitializationComplete and VerifiedNode messages

### PR DESCRIPTION
**Pull request by bobjacobsen**
See Issue #254 for background

### CODE REVIEW

File: src/org/openlcb/OpenLcb.java [hunk 1]
```diff
@@ -12,9 +12,11 @@ public interface OpenLcb {
      */
      
     static final int MTI_INITIALIZATION_COMPLETE     = (int)MessageTypeIdentifier.InitializationComplete.mti();
+    static final int MTI_INITIALIZATION_COMPLETE_SIMPLE = (int)MessageTypeIdentifier.InitializationCompleteSimple.mti();
     
     static final int MTI_VERIFY_NID                  = 0x10A4;
     static final int MTI_VERIFIED_NID                = 0x10B0;
+    static final int MTI_VERIFIED_NID_SIMPLE         = 0x10B1;
```
Comment (balazsracz) [inline src/org/openlcb/OpenLcb.java:19]: these values are wrong. I believe they are from pre-2012 when the MTI numbers were different.
I hope they are not used in the code...

File: src/org/openlcb/InitializationCompleteSimpleMessage.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+ * with the SImple Protocol modifier set
+ *
+ * @author  Bob Jacobsen   Copyright 2024
+ * @version $Revision$
+ */
+@Immutable
+@ThreadSafe
+public class InitializationCompleteSimpleMessage extends InitializationCompleteMessage {
+    
+    public InitializationCompleteSimpleMessage(NodeID source) {
```
Comment (balazsracz) [inline src/org/openlcb/InitializationCompleteSimpleMessage.java:18]: if we do it with a new class, we should add a function `isSimpleProtocolSubset()` that is false in the base class and true here.

Alternatively, we could avoid making new classes and instead take the "isSimple" bit on the constructor. The producer/consumer identified messages do that; when one m,essage has variants that are different in some minor ways.
I don't have a strong preference.

File: src/org/openlcb/can/MessageBuilder.java [hunk 3]
```diff
@@ -239,6 +239,11 @@ List<Message> processFormat1(CanFrame f) {
             case InitializationComplete:
                 retlist.add(new InitializationCompleteMessage(source));
                 return retlist;
+
+            case InitializationCompleteSimple:
```
Comment (balazsracz) [inline src/org/openlcb/can/MessageBuilder.java:243]: could you add unit tests into https://github.com/openlcb/OpenLCB_Java/blob/master/test/org/openlcb/can/MessageBuilderTest.java

We need two tests per message type: one rendering messages into can frames and one p[arsing a can frame into the message.

File: src/org/openlcb/OpenLcb.java (hunk 1 above)
Comment (bobjacobsen) [reply src/org/openlcb/OpenLcb.java:19]: As near as I can tell these values and the associated `getMTI()` method aren't used anywhere.  I'll remove them at some point.

File: src/org/openlcb/InitializationCompleteSimpleMessage.java (hunk 2 above)
Comment (bobjacobsen) [reply src/org/openlcb/InitializationCompleteSimpleMessage.java:18]: The next PR will do this with a property on the messages and `hasSimpleProtocol()`

**Comment by bobjacobsen** [comment]
Closed in favor of the solution in #264

**Pull request closed without merging**